### ISSUE TITLE
hourly data

### ISSUE THREAD

**Issue by ghost** [nFrechen/RgetDWDdata#3]
Hi nFrechen ;)

thank you for the comfortable package!

It would be great if you could also download hourly data from

ftp://ftp-cdc.dwd.de/pub/CDC/observations_germany/climate/hourly/

Best wishes 
Christof

**Comment by nFrechen** [comment]
We already worked on this. If you install the developer version you can test it:

`install_github("nFrechen/RgetDWDdata", ref="feature/hourly_data")`

Currently I can't remember how far we got with this and whether it still works. Tell me about your test results!

Greetings,
Nanu Frechen

**Comment by ghost** [comment]
thank you very much! There seems to be a problem

```
getDWDhourly("Cottbus", historisch = F,  Parameter = "precipitation")
Error in getDWDhourly("Cottbus", historisch = F, Parameter = "precipitation") : unused argument (Parameter = "precipitation")
```

BUT the code direct works fine! (with \r\n instead of \n on my windows machine)
https://github.com/nFrechen/RgetDWDdata/blob/feature/hourly_data/R/getDWDhourly.R

Greetings
Christof

**Comment by nFrechen** [comment]
Yes, I remember changing the \n to \r\n in the `getDWDdata()` function. I think your error appears because there is a second file with a `getDWDhourly()` function in it: `getDWDhourly_erweiterung.R`. This one has no argument `Parameter =`. I guess the file `getDWDhourly_erweiterung.R` is something old that has to be deleted.
My colleague @TobRecke wrote this function, so I don't have as much insight in it as into my own functions. I'll ask him or read into it myself to do some changes. If you are able to do changes you are welcome to make a pull request!

**Comment by nFrechen** [comment]
There is a new version you can test. @TobRecke had already written it but didn't upload it yet.

What does not always work is downloading data by station name (in contrast to station id). That is because the files where we look up the station names and connect a station id to it are not in a unique format. Some of them can't be read by our function. I contacted Frank Kaspar from DWD about it.

Please tell us what works and what not. Especially about the \r\n to \n thing. I cannot test this on my machine...

**Comment by nFrechen** [comment]
I solved the problem with the different formatting in the Station_id in the stations overview files in commit 6b86683f2b4e78038a2f702b7a7c0429922b5eae. Also there was a problem with some of the file names having a different format. So I introduced a different way to resolve file name from station id in commit 58e107a1f0403ecdb7268155c5ffff38f2ffd5ec.

We might be ready to put the `getDWDhourly` function into the master branch, but maybe we all do some further tests before. @TobRecke what do you think?

**Comment by TobRCB** [comment]
It sounds and looks smart..
Maybe i'll come around to the fablab for a chat during the next weeks..

2016-08-03 19:19 GMT+02:00 Nanu Tobias Frechen notifications@github.com:
